### ISSUE TITLE
Many errors on unpack CAB archive

### ISSUE THREAD

**Issue by ikrivosheev** [mdsteele/rust-cab#14]
@mdsteele hello! thank you for the great library! I have cab archive (get from MSI): [python.zip](https://github.com/mdsteele/rust-cab/files/8825947/python.zip)

I have two problems:

1. Unpacking is very slow
2. A lot errors:
```
w9xpopen.exe unpacker error=Custom { kind: Other, error: ChunkTooLong }
README.txt unpacker error=Custom { kind: Other, error: ChunkTooLong }
NEWS.txt unpacker error=Custom { kind: Other, error: OverreadBlock }
LICENSE.txt unpacker error=Custom { kind: Other, error: InvalidBlock(0) }
python.exe unpacker error=Custom { kind: Other, error: ChunkTooLong }
pythonw.exe unpacker error=Custom { kind: Other, error: ChunkTooLong }
python27.dll unpacker error=Custom { kind: Other, error: OverreadBlock }
...
```

**Comment by mdsteele** [comment]
Thanks.  It looks like that CAB file is using `Lzx(21)` compression, and those errors seem to be coming from the `lzxd` crate (see https://github.com/Lonami/lzxd/blob/ba70d7f8292ce840267b1ab89d44a9685c72ba1f/src/lib.rs#L90).  The `cab` crate is already using the latest available version of `lzxd` (v0.1.4).  So probably you'll need to report these issues there, sorry.

**Comment by ikrivosheev** [comment]
@mdsteele thank you! I created issue
